templates/display-sp-tpl.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * This is a simple example of template with table of SPs
+ *
+ * @author Dominik Baránek <0Baranek.dominik0@gmail.com>
+ */
+
+$this->data['header'] = '';
+$this->includeAtTemplateBase('includes/header.php');
+
+$attrNames = $this->data['attrNames'];
+$facilitiesWithAttributes = $this->data['facilitiesWithAttributes'];
+
+$columns = '[';
+$columns .= '{label: "Id", type: "number"},';
+$columns .= '{label: "Name", type: "string"},';
+$columns .= '{label: "Description", type: "string"},';
+if (!empty($attrNames)) {
```

Comment:
I think that is bad idea to check only if type is one of Integer or Boolean and if not that the type is automatically string because from perun you can get for example ArrayList or LinkedHashMap